docs/rundown.rst

```diff
@@ -34,6 +34,44 @@ and are ready to move on!
 .. _GPG: https://www.gnupg.org/
 .. _Red Hat "Enable Sysadmin" blog post: https://www.redhat.com/sysadmin/creating-gpg-keypairs
 
+How to Access the Ansible Sign CLI Utility:
+===========================================
+Clone ansible-sign as an entirely new repo (This is typically placed within an existing AWX repo).
+
+``$ git clone git@github.com:ansible/ansible-sign``
+
+There are certainly alternative methods for closing a repo, absolutely use whichever method you prefer. (You might use github's https cloning instead of the git@ syntax, which is SSH.)
+
+Then once you have that cloned, you can:
+``$ cd ansible-sign``
+
+Then:
+``$ pip install -e .``
```

Comment:
This is a good developer workflow for people who want to work on ansible-sign for any reason. For end users, we probably want them to install it from PyPI (or from dnf if they are using RHEL/AAP?).

So all of these lines can effectively just become:
> Run `pip install ansible-sign`.

... for end users.

Reply:
Thank you so much! I will revise this bit accordingly!

Reply:
I have this saved in my notes for if we want to make a "How to contribute" page later. I am updating as per your instructions.